### DISCUSSION TITLE
Jgroups udp-cluster not working anymore and still getting 'Query timed out after PT2S'

### DISCUSSION THREAD

**Discussion by MikaMLehtonen** [glowroot/glowroot#1063] [General]
Clustering broke after upgrading from 0.14.0 to 0.14.1, getting error:

SEVERE: Allocate exception for servlet [org.glowroot.central.GlowrootServlet]
javax.management.InstanceAlreadyExistsException: org.infinispan:type=CacheManager,name="DefaultCacheManager",component=CacheManager
and in browser:
Message org.infinispan.manager.EmbeddedCacheManagerStartupException: ISPN000034: The 'org.infinispan' JMX domain is already in use
Nothing seems to help..

Also this 'Query timed out after PT2S' is still bothering.

Have to downgrade back to some 0.13 or maybe 0.14. This has been a very great tool, but can't spend any time anymore.
Maybe it's about our clustering configuration? QA-version (non-clustered) seems to be working just fine, although there's no real traffic there.

Mika Lehtonen

**Comment by nowheresly** [comment]
Query timed out after PT2S should be fixed with version 0.14.1. Feel free to open an issue with a stack trace if it is not the case.

The error you get about JMX domain seems to related to the fact that your host already have an infinispan started on it. (https://stackoverflow.com/questions/10295751/org-infinispan-jmx-jmxdomainconflictexception-domain-already-registered-org-inf)
